keps/sig-apps/2255-pod-cost/README.md

```diff
@@ -55,163 +54,183 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
-
+Unit tests. 
 
 ### Rollout, Upgrade and Rollback Planning
 
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
-  - As the feature is a simple annoation, the worst what could happen is that either the annotation is lost or ignored. In the worst case, a pod with a higher priority gets deleted before a pod with a lower priority.
-
+It shouldn't impact already running workloads. This is an opt-in feature
+since users need to explicitly set the annotation.
 
 * **What specific metrics should inform a rollback?**
-  - None
 
 
 * **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
-  - Was tested. Behaviour change in both directions, as expected.
-
+We can do manual testing.
 
 * **Is the rollout accompanied by any deprecations and/or removals of features,
   APIs, fields of API types, flags, etc.?**
-  - No. However, the exact same pod annotation string cannot be used for any other purposes.
-
+No.
 
 ### Monitoring requirements
 
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can an operator determine if the feature is in use by workloads?**
-  - Search for pod annotations with the exact same pod-cost annotation string.
 
 
 * **What are the SLIs (Service Level Indicators) an operator can use to
   determine the health of the service?**
-  - A pod with a lower pod-cost annotation in a Deployment gets deleted first on a scale-down event.
 
 
 * **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
-  - All pods with a lower pod-cost annotation in a Deployment are deleted first on a scale-down event.
 
 * **Are there any missing metrics that would be useful to have to improve
   observability if this feature?**
-  - N/A
 
 ### Dependencies
 
 _This section must be completed when targeting beta graduation to a release._
 
 * **Does this feature depend on any specific services running in the cluster?**
-  - The feature requires the existing of the kube-controller-manager and the ability and permissions to set pod annotations.
-
+No.
 
 ### Scalability
 
-_For alpha, this section is encouraged: reviewers should consider these questions
-and attempt to answer them._
-
-_For beta, this section is required: reviewers must answer these questions._
-
-_For GA, this section is required: approvers should be able to confirms the
-previous answers based on experience in the field._
-
 * **Will enabling / using this feature result in any new API calls?**
-  - Whenever the application decides, that a change in pod-cost is needed for a replica, it will send out an API request and set the appropriate pod annotation(s).
+  - No, not the feature itself. However, users will want to deploy an external controller 
+    that updates the pod-deletion-cost, documentation should stress that update frequency
+    to be coarse grained.
```

Comment:
Add "no" for the 4th question (line 247).

Reply:
done.